components/typeform/actions/list-forms/list-forms.mjs

```diff
@@ -36,24 +37,19 @@ export default {
       description: "Retrieve typeforms for the specified workspace.",
```

Comment:
**-Minor-**
Move the [prop definition](https://pipedream.com/docs/components/guidelines/#prop-definitions) for `workspaceId` from this file to `typeform.app.mjs` if you think it might be reused in future components?